docs/source/executing-operations/defer-support.mdx

```diff
@@ -1,13 +1,13 @@
 ---
-title: Apollo Router Support for @defer
+title: GraphOS Router Support for @defer
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)